pkg/proto/configuration/bb_runner/bb_runner.proto

```diff
@@ -31,4 +31,7 @@ message ApplicationConfiguration {
   // Optional helper process for resolving /tmp.
   buildbarn.configuration.grpc.ClientConfiguration
       temporary_directory_installer = 6;
+
+  // Chroot into the input root to run commands
```

Comment:
Could you extend this comment to explain why/when you'd want this? Also a perfect moment to promote Buildstream!

"This option can be used if the input root contains a full userland installation. This feature is used by the BuildStream build system."